modules/nf-core/trimgalore/environment.yml

```diff
@@ -0,0 +1,7 @@
+channels:
+  - conda-forge
+  - bioconda
+
```

Comment:
We should remove this empty line upstream, but low low low priority.
I mean, if you have nothing else to do, just go have a coffee instead, this should be automatically linted and fixed